src/chisel/lib/uart/Uart.scala

```diff
@@ -190,13 +194,36 @@ class Echo(frequency: Int, baudRate: Int) extends Module {
   tx.io.channel <> rx.io.channel
 }
 
-class UartMain(frequency: Int, baudRate: Int) extends Module {
+/** Combine RGB and Input/Output. */
+class LEDInput(frequency: Int, baudRate: Int) extends Module {
```

Comment:
Please copy this over to your application file, i.e. `src/design/LEDToggle.scala `.

Reply:
And restore the old `UartMain`.